plugins/test/unit/plugins/importers/yum/test_sync.py

```diff
@@ -573,9 +608,15 @@ def test_metadata_revision_zero(self, mock_metadata_files):
         self.assertTrue(self.reposync.skip_repomd_steps is False)
         self.assertEqual(mock_metadata_instance.download_metadata_files.call_count, 1)
 
+    @mock.patch('pulp_rpm.plugins.importers.yum.sync.repo_controller.check_perform_full_sync',
+                autospec=True)
     @mock.patch.object(metadata, 'MetadataFiles', autospec=True)
-    def test_metadata_feed_override(self, mock_metadata_files):
-        """with feed override is set, a full sync should be performed"""
+    def test_metadata_feed_override(self, mock_metadata_files, mock_check_sync):
```

Comment:
i think this test can be removed because you removed that condition from the code `self.config.override_config.get(importer_constants.KEY_FEED)`